imports/client/components/AnnouncementsPage.tsx

```diff
@@ -53,7 +54,7 @@ const AnnouncementForm = ({ huntId }: { huntId: string }) => {
   const postAnnouncement = useCallback(() => {
     if (message) {
       setSubmitState(AnnouncementFormSubmitState.SUBMITTING);
-      Meteor.call('postAnnouncement', huntId, message, (error?: Error) => {
+      post.call({ huntId, message }, (error) => {
```

Comment:
Reflecting on this naming scheme -- I wonder if having these reduce to a single verb is going to make it harder to track semantically.  If this were Go and you always had to invoke things by `module.verb()`, then it'd be less ambiguous, but I'm thinking about what happens if we follow this to its logical conclusion and wind up with `Hunt.methods.create` and `Puzzle.methods.create` and `Tags.methods.create` and all of them want to be called `create` in the same file.

Maybe we should aim to name things as `verbNoun` instead of just `verb`?  That adds a little more context as the usage site.  If we're doing that, I'd also consider if it makes sense to name the method file `postAnnouncement` for consistency, though that makes the imports do a bit of repetition.

I guess another option is we wrap up the calls on objects onto some facade object and then maybe it's `AnnouncementMethods.post.call`.

Reply:
Yeah I think I agree. I'll switch to that.